cmd/status.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2016-2018 Genome Research Limited
+// Copyright © 2016-2021 Genome Research Limited
```

Comment:
2016-2018, 2021 ?

Reply:
No, I checked the logs and it was updated in 2019 and 2020 as well.